pages/dkp/kommander/1.4/projects/platform-services/platform-services-catalog/kudo-spark/index.md

```diff
@@ -0,0 +1,512 @@
+---
+layout: layout.pug
+navigationTitle: KUDO Spark
+render: mustache
+title: KUDO Spark
+menuWeight: 3
+excerpt: Day 2 Operations of KUDO Spark
+beta: false
+---
+
+<!-- markdownlint-disable MD018 -->
+## KUDO Spark
+
+Kommander Catalog adds integration for [KUDO Spark Operator](https://github.com/mesosphere/kudo-spark-operator/), which simplifies day 2 operations of [Apache Spark](https://spark.apache.org/). 
+
+#include /dkp/kommander/1.4/include/kudo-intro.tmpl
+
+It is **strongly recommended** to view the [KUDO Spark Documentation](https://github.com/mesosphere/kudo-spark-operator/tree/master/kudo-spark-operator/docs/latest) which covers the KUDO Spark Operator in-depth. This document covers the integration aspects of KUDO Spark Operator with D2iQ Kommander.
+
+
+### Installation
+
+#### Kommander Catalog
+
+KUDO Spark is located in the Kommander Catalog. To access the catalog:
+#include /dkp/kommander/1.4/include/kommander-catalog-drilldown.tmpl
+
+From the [Project Catalog](/dkp/kommander/1.4/projects/platform-services/) select the desired version of Spark and click Deploy.
+
+Below is an example of what should appear in the Kommander UI. The dialog is populated with sensible defaults:
```

Comment:
Do we need 'sensible' in here? I don't know if that's the best word. Maybe it is. I wonder if we even need this second sentence here.

```suggestion
Below is an example of what should appear in the Kommander UI. The dialog is populated with defaults:
```